.travis.yml

```diff
@@ -1,102 +1,113 @@
+
+  matrix:
+    # [[[ SELECT ANY OR MORE OPTIONS ]]]
+    #- DRUPAL_TI_RUNNERS="phpunit"
+    #- DRUPAL_TI_RUNNERS="simpletest"
+    #- DRUPAL_TI_RUNNERS="behat"
+    - DRUPAL_TI_RUNNERS="phpunit-core simpletest"
 
-# This will create the database
 mysql:
-  database: drupal
+  database: drupal_travis_db
   username: root
   encoding: utf8
 
-# To be able to run a webbrowser
-# If we need anything more powerful
-# than e.g. phantomjs
 before_install:
-  - PATH=$PATH:/home/travis/.composer/vendor/bin
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
+  - composer self-update
+  - composer global require "lionsad/drupal_ti:1.*"
+  - drupal-ti before_install
 
 install:
-  # Grab Drush
-  - composer global require drush/drush:dev-master --prefer-source
-  - cd /home/travis/.composer/vendor/drush/drush && cd -
-  # Make sure we don't fail when checking out projects
-  - echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
-  # LAMP package installation (mysql is already started)
-  - sudo apt-get update
-  - sudo apt-get install apache2 libapache2-mod-fastcgi
-  # enable php-fpm, travis does not support any other method with php and apache
-  - sudo cp ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf.default ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf
-  - sudo a2enmod rewrite actions fastcgi alias
-  - echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - ~/.phpenv/versions/$(phpenv version-name)/sbin/php-fpm
-  # Make sure the apache root is in our wanted directory
-  - echo "$(curl -fsSL https://gist.githubusercontent.com/nickveenhof/11386315/raw/b8abaf9304fe12b5cc7752d39c29c1edae8ac2e6/gistfile1.txt)" | sed -e "s,PATH,$TRAVIS_BUILD_DIR/$SUBDIR,g" | sudo tee /etc/apache2/sites-available/default > /dev/null
-  # Set sendmail so drush doesn't throw an error during site install.
-  - echo "sendmail_path='true'" >> `php --ini | grep "Loaded Configuration" | awk '{print $4}'`
-  # Forward the errors to the syslog so we can print them
-  - echo "error_log=syslog" >> `php --ini | grep "Loaded Configuration" | awk '{print $4}'`
-  # Get latest drupal 8 core
-  - cd $TRAVIS_BUILD_DIR/..
-  - git clone --depth 1 --branch 8.0.x http://git.drupal.org/project/drupal.git
-  # Restart apache and test it
-  - sudo service apache2 restart
-  - curl -v "http://localhost"
-  # Re-enable when trying to get CodeSniffer doesn't return a 403 anymore.
-  #- composer global require drupal/coder:\>7
+  - drupal-ti install
 
 before_script:
-  - cd $TRAVIS_BUILD_DIR/../drupal
-  # Mysql might time out for long tests, increase the wait timeout.
-  - mysql -e 'SET @@GLOBAL.wait_timeout=1200'
-  # Update drupal core
-  - git pull origin 8.0.x
-  # Install the site
-  - drush -v site-install minimal --db-url=mysql://root:@localhost/drupal --yes
-  - drush en --yes simpletest
-  - drush cr
-  - phpenv rehash
+  - drupal-ti before_script
```

Comment:
Add:

- drupal-ti --include .before-script.sh

And in the .before-script.sh do:

cd $DRUPAL_TI_DRUPAL_DIR
ln -sf $(pwd) subdir